docs/resources/azure_redis_cache.md

```diff
@@ -0,0 +1,117 @@
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md).
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure).
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+`resource_group` and `name` must be given as a parameter.
+```ruby
+describe azure_redis_cache(resource_group: 'MyResourceGroup', name: 'inspec-compliance-redis-cache') do
+  it                                      { should exist }
+  its('name')                             { should cmp 'inspec-compliance-redis-cache' }
+  its('type')                             { should cmp 'Microsoft.Cache/Redis' }
+  its('sku.name')                         { should cmp 'Standard' }
+  its('sku.family')                       { should cmp 'C' }
+  its('location')                         { should cmp 'southcentralus' }
+end
+```
+```ruby
+describe azure_redis_cache(resource_group: 'MyResourceGroup', name: 'inspec-compliance-redis-cache') do
+  it  { should exist }
+end
+```
+## Parameters
+
+| Name                            | Description                                                                      |
+|---------------------------------|----------------------------------------------------------------------------------|
+| resource_group                  | Azure resource group that the targeted resource resides in. `MyResourceGroup`    |
+| name                            | Name of the Azure Redis Cache to test.                                           |
+
+The parameter set should be provided for a valid query:
+- `resource_group` and `name`
+
+## Properties
+
+| Property                      | Description                                                      |
+|-------------------------------|------------------------------------------------------------------|
+| id                            | Resource ID.                                                     |
+| name                          | Redis Cache Name.                                       |
+| location                      | Redis Cache Location.                                   |
+| type                          | Resource type.                                                   |
+| tags                          | Resource tags.                                                   |
+| properties.sku.name           | The unique name of the SKU, such as 'P3'.                        |
```

Comment:
The values in the tables that start with 'properties' is not inclueded in the URI Parameters tables of this link: https://docs.microsoft.com/en-us/rest/api/redis/redis/get

Reply:
this table refers to the [properties](https://docs.microsoft.com/en-us/rest/api/redis/redis/get#redisresource) section in the doc and not the URI parameters section

Reply:
It is the properties that aren't matching. For example, SkuName: "The type of Redis cache to deploy. Valid values: (Basic, Standard, Premium)" I think it's just the Sku properties that don't match though.

Reply:
yeah, you are right.. my bad I just updated it ..